reshenie.java

```diff
@@ -0,0 +1,11 @@
+public class Main {
+    public static void main(String[] args) {
+        ball ball1 = Lottery.getRandomBall();
+        ball ball2 = Lottery.getRandomBall();
+        ball ball3 = Lottery.getRandomBall();

```

Comment:
The class name `ball` should be capitalized as `Ball` to follow Java naming conventions for classes. This change should be reflected in all instances where `ball` is used.